modules/installation-configuration-parameters.adoc

```diff
@@ -473,6 +473,18 @@ Optional installation configuration parameters are described in the following ta
 |A PEM-encoded X.509 certificate bundle that is added to the nodes' trusted certificate store. This trust bundle may also be used when a proxy has been configured.
 |String
 
+|`capabilities`
+|Controls the installation of optional core cluster components.
+|String array
+
+|`capabilities.baselineCapabilitySet`
+|Selects an initial set of optional capabilities to enable. Valid values are `None`, `v4.11` and `vCurrent`. `v4.11` enables the `baremetal` Operator, the `marketplace` Operator, and the `openshift-samples` content. `vCurrent` installs the recommended set of capabilities for the current version of {product-title}. The default value is `vCurrent`.
+|String
+
+|`capabilities.additionalEnabledCapabilities`
```

Comment:
@bscott-rh  Admittedly, I am a bit removed from this, but it is unclear me to when you would use `capabilities.additionalEnabledCapabilities`.  Is this a case of setting `capabilities.baselineCapabilitySet` to `None` and then using `capabilities.additionalEnabledCapabilities` to optional enable 1 or 2 of the optional capabilities?

Reply:
That's correct. There's not much reason to do it right now, but I believe the set of capabilities will be expanded each release so future users will have more control over which subsets of features to enable.